assets/js/pages/SaptuneDetails/SaptuneTuningState.jsx

```diff
@@ -9,7 +10,12 @@ function SaptuneTuningState({ state }) {
     case 'not compliant':
       return (
         <div className="flex">
-          <HealthIcon health="critical" />
+          <Tooltip
+            content="Run `saptune note verify` in the host for further details"
```

Comment:
I don't think markdown works here as it is. You would need to wrap it up, something like:
```
<ReactMarkdown className="markdown" remarkPlugins={[remarkGfm]}>
Run `saptune note verify` in the host for further details
 </ReactMarkdown>
```